fileWithPRComments.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
```

Comment:
Codacy found an issue: [Move the invocation into the parens that contain the function.](https://app.staging.codacy.org/gh/stagingcodacytestuser/CheckPrCommentsPublicProjectStaging/pullRequest?prid=11541)